modules/ROOT/pages/Development/Satisfactory/CustomLevels.adoc

```diff
@@ -0,0 +1,64 @@
+* {blank}
++
+Map Name::
+  A field to define the name of the level as it should show in the main menu.
+* {blank}
++
+Map Description::
+  This field determins the map description as it should show in the main menu.
+* {blank}
++
+Map Icon::
+  Defines the image to show for the map in the main menu.
+* {blank}
++
+Starting Locations::
+  Created as an array, this defines the different possible locations the pioneer can spawn in. Refer to the Defining Spawn Points section on how to use this array.
+
+== Starting Locations Array
+
+The starting locations array, part of the SMLGameMapData asset defines where the player is able to start at.
+
+    TODO: Figure out how to create/add Starting Location objects, and create the array
+
+== Creating a Level [.title-ref]#(LevelCreation)#
+
+This section is meant to be closer to a tutorial on implementing everything mentioned on this page to add your own level.
+
+The process of creating a level is fairly simple, with these docs meant to guide you through, using the info above as reference.
+
+The first step to creating a level is, of course, creating a level, not to be confused with creating a level, of course. The recommended way to do this would be to open the Example Mod folder, and copy the Maps folder into your own plugin, which you can then modify. The docs will asume, from this point, that is what you did.
+
+To begin, open the new copied folder, and open the SMLGameMapData asset, which, when copied will posses the name GameMapAsset_ExampleLevel. Renaming this asset, as well as the ExampleLevel asset is highly recommended, to avoid confusion with the original files from the Example Mod folder.
+
+Next, open the GameMapAsset, and change the following fields.
+
+Map Asset to match the new Level file you have inside your mods plugin. After you do this, change the Map Name to the name you would like to show inside the game, then set the Map Description to the description you would like to show in game, then set the Map Icon to an icon you would like (the recommended image size to use for the Map Icon is 1024x1024). After which, set the Starting Location array by doing the following:
+
+    TODO: Starting Location Array Guide
+
+Once these steps are done, you should be good to go! Simply Alpakit your mod into the game, and open the New Game tab, and, assuming everything was done correctly, you should have your custom map.
+
+== Adding Objects To a Custom Level
+
+If you have a custom level loading, odds are you want some objects in there, this list details how to do just that.
+
+To add items, simply open the Blueprints/ folder. This should be included with the folder you copied from the Example Mod. Then, find what you want to place, in this case, let's do an ore node, which would be Blueprints/Node_ExampleDynamic, and drag it where you want on the map. While it might not look right in the Editor, it will be replaced with the proper model and texture in the game.

```

Comment:
I suggest to use "All/FactoryGame/Resource/BP_ResourceNode" for example of ore node.
Node_ExampleDynamic needs bool value of Session Settings "ExampleMod.Level.SpawnNodes" and also they don't show ping when you use "scan" in the game.

BP_ResourceNode needs write extra steps but it's easy to place them when you can see them.

Reply:
Why was this conversation resolved? I don't see any commits after the comment was made to change it

Reply:
Also, the example node is added by Example Mod and as such won't exist if that mod isn't loaded

The scan issue is caused by any runtime created node, so as long as the node you add is part of the level and not spawned at runtime, they should show on the scanner.

Reply:
(Almine's changes fixed this and were merged in later)